### ISSUE TITLE
Remove non-BSL files

### ISSUE THREAD

**Issue by hkaiser** [STEllAR-GROUP/hpx#5089]
We currently have several leftover files that are not licensed under the BSL:

- https://github.com/STEllAR-GROUP/hpx/blob/master/plugins/parcelport/verbs/rdma/RdmaCompletionChannel.h.good
- https://github.com/STEllAR-GROUP/hpx/blob/master/examples/quickstart/EasyBMP

We should either re-license or remove those files from the repository

**Comment by msimberg** [comment]
Good catch! That verbs file doesn't even look like it's used anywhere. On top of that the verbs parcelport has been deprecated since 1.4.0 and could be removed completely.

Unsure how dear/useful the fractals/EasyBMP example is.

**Comment by hkaiser** [comment]
> Unsure how dear/useful the fractals/EasyBMP example is.

Let's remove these examples.
